packages/mobile/screens/SignIn.tsx

```diff
@@ -0,0 +1,203 @@
+                setErrors({ server: error[0] });
+            });
+    };
+
+    return (
+        <Container>
+            <Header>
+                <Image resizeMode={'cover'}
+                    source={require('../assets/logo-lithium.png')}
+                />
+                <WelcomeText>{'Welcome to Lithium KB'}</WelcomeText>
+            </Header>
+            <Title>Login</Title>
+            <Formik
+                initialValues={{ email: '', password: '' }}
+                validationSchema={LoginSchema}
+                onSubmit={submition}
+            >
+                {({ values: { email, password }, handleChange, handleSubmit, errors, touched, handleBlur, }:
+                    { handleSubmit: any, values: any, handleChange: any, errors: any, touched: any, handleBlur: any }) => (
+                        <React.Fragment>
+
+                            {touched.email && errors?.email?.length &&
+                                <Text style={{ fontSize: 10, color: 'red' }}>{errors.email}</Text>
+                            }
+                            <Input>
+
+                                <Field
+                                    id="email"
+                                    name="email"
+                                    type="email"
+                                    component={TextInputCust}
+                                    value={email}
+                                    placeholder="Email..."
+                                    placeholderTextColor="#003f5c"
+                                    onChangeText={handleChange('email')}
+                                    error={touched["email"] && errors["email"]?.length > 0}
+                                    onBlur={handleBlur('email')}
+                                />
+                            </Input>
+
+                            {touched.password && errors?.password?.length &&
+                                <Text style={{ fontSize: 10, color: 'red' }}>{errors.password}</Text>
+                            }
+                            <Input >
+
+                                <Field
+                                    id="password"
+                                    name="password"
+                                    type="password"
+                                    component={TextInputCust}
+                                    value={password}
+                                    secureTextEntry
+                                    placeholder="Password..."
+                                    placeholderTextColor="#003f5c"
+                                    onChangeText={handleChange('password')}
+                                    error={touched["password"] && errors["password"]?.length > 0}
+                                    onBlur={handleBlur('password')}
+                                />
+                            </Input>
+
+                            <ForgotText >{'Forgot de password?'}</ForgotText>
```

Comment:
the*

In the design it's "Forgot password?" so please  double check the designs